tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/BundleFactoryDelegate.java

```diff
@@ -53,11 +55,24 @@ public BundleFactoryDelegate()
     {
         IModule[] retval = new IModule[0];
 
-        ILiferayProject liferayProject = LiferayCore.create( project );
-
-        if( liferayProject instanceof IBundleProject )
+        try
+        {
+            ILiferayProject liferayProject = LiferayCore.create( project );
+
+            if( liferayProject instanceof IBundleProject )
```

Comment:
So maven projects that are "Web" projects IWebProjects should not be tried as bundle projects.  But there are maven projects (have maven nature) that are "bundle" projects only, (not web projects).